### PR TITLE
added LiveEventClosed messageType and args params to Host constructor

### PR DESCRIPTION
**Pull request by TamirFolloze**

### CODE REVIEW

File: src/host/Widget.ts [hunk 1]
```diff
@@ -56,4 +61,9 @@ export class Widget {
         return this.messanger.on(type, handler);
     }
 
+    liveEventClosed() {
+        console.log("messanger send liveEventClosed")
```
Comment (itamargolan) [inline src/host/Widget.ts:65]: remove

File: src/host/Widget.ts [hunk 2]
```diff
@@ -56,4 +61,9 @@ export class Widget {
         return this.messanger.on(type, handler);
     }
 
+    liveEventClosed() {
```
Comment (itamargolan) [inline src/host/Widget.ts:64]: doesn't the child need to know how to publish it too?

**Review by itamargolan — CHANGES_REQUESTED** [review]

File: src/host/Widget.ts (hunk 1 above)
Comment (TamirFolloze) [reply src/host/Widget.ts]: done

File: src/host/Widget.ts (hunk 2 above)
Comment (TamirFolloze) [reply src/host/Widget.ts:64]: added participantLeftMeeting messageType to child

File: src/child/Child.ts [hunk 3]
```diff
@@ -53,6 +53,10 @@ export class Child extends ChildMessanger{
         this.send(MessageType.UrlChanged, url);
     }
 
+    public participantLeftMeeting() {
```
Comment (itamargolan) [inline src/child/Child.ts:56]: why not closeLiveEvent?
for example, seller dashboard sends "setList" and not technologyChanged

**Review by itamargolan — CHANGES_REQUESTED** [review]

File: src/child/Child.ts (hunk 3 above)
Comment (TamirFolloze) [reply src/child/Child.ts]: done

**Review by itamargolan — APPROVED** [review]

**Pull request merged**